microsoft-edge/webview2/concepts/working-with-local-content.md

```diff
@@ -18,12 +18,21 @@ In addition to loading remote content, content can also be loaded locally into W
 
 These approaches are described below.
 
```

Comment:
DR's question 4 from main page of PR:

tbd

Reply:
In writers thread I'm asking whether to go towards eg:

```
A hyphen in the following table indicates Not Supported:

| Scenario | Navigating to a file URL | Navigating to an HTML string | Virtual host name mapping | `WebResourceRequested` |
| --- | --- | --- | --- | --- |
| Origin-based DOM APIs | Supported | - | Supported | Supported |
| DOM APIs requiring secure context | - | - | Supported | Supported |
| Dynamic content | - | Supported | - | Supported |
| Additional web resources | Supported | - | Supported  | Supported |
| Additional web resources resolved in WebView2 process | Supported | - | Supported | - |

For details, see the following sections.
```

Reply:
Ok then should we resolve this comment?

Reply:
yes